### ISSUE TITLE
Install fails with msg: Job for openshift-node.service failed... can't find docker.sock

### ISSUE THREAD

**Issue by etsauer** [openshift/openshift-ansible#515]
```
TASK: [openshift_node | Start and enable openshift-node] ********************** 
failed: [node3.example.com] => {"failed": true}
msg: Job for openshift-node.service failed. See 'systemctl status openshift-node.service' and 'journalctl -xn' for details.

failed: [node2.example.com] => {"failed": true}
msg: Job for openshift-node.service failed. See 'systemctl status openshift-node.service' and 'journalctl -xn' for details.

failed: [master.example.com] => {"failed": true}
msg: Job for openshift-node.service failed. See 'systemctl status openshift-node.service' and 'journalctl -xn' for details.

failed: [node4.example.com] => {"failed": true}
msg: Job for openshift-node.service failed. See 'systemctl status openshift-node.service' and 'journalctl -xn' for details.
```

Checking why the services fail to start, I get failures finding docker running, though the docker service is active;

```
[root@master ~]# systemctl status openshift-node -l
openshift-node.service - OpenShift Node
   Loaded: loaded (/usr/lib/systemd/system/openshift-node.service; enabled)
  Drop-In: /usr/lib/systemd/system/openshift-node.service.d
           └─openshift-sdn-ovs.conf
   Active: failed (Result: exit-code) since Wed 2015-08-26 19:07:19 PDT; 7min ago
     Docs: https://github.com/openshift/origin
  Process: 61805 ExecStart=/usr/bin/openshift start node --config=${CONFIG_FILE} $OPTIONS (code=exited, status=255)
 Main PID: 61805 (code=exited, status=255)

Aug 26 19:07:18 master.p1.rhc-ose.labs.redhat.com systemd[1]: Starting OpenShift Node...
Aug 26 19:07:19 master.p1.rhc-ose.labs.redhat.com openshift-node[61805]: I0826 19:07:19.249596   61805 start_node.go:177] Starting an OpenShift node, connecting to https://master.p1.rhc-ose.labs.redhat.com:8443
Aug 26 19:07:19 master.p1.rhc-ose.labs.redhat.com openshift-node[61805]: I0826 19:07:19.466471   61805 docker.go:295] Connecting to docker on unix:///var/run/docker.sock
Aug 26 19:07:19 master.p1.rhc-ose.labs.redhat.com openshift-node[61805]: I0826 19:07:19.466671   61805 start_node.go:263] Starting OpenShift node master.p1.rhc-ose.labs.redhat.com (v3.0.1.0-528-g8c2fe51)
Aug 26 19:07:19 master.p1.rhc-ose.labs.redhat.com openshift-node[61805]: F0826 19:07:19.686306   61805 node.go:85] ERROR: Docker could not be reached at unix:///var/run/docker.sock.  Docker must be installed and running to start containers.
Aug 26 19:07:19 master.p1.rhc-ose.labs.redhat.com openshift-node[61805]: dial unix /var/run/docker.sock: no such file or directory
Aug 26 19:07:19 master.p1.rhc-ose.labs.redhat.com systemd[1]: openshift-node.service: main process exited, code=exited, status=255/n/a
Aug 26 19:07:19 master.p1.rhc-ose.labs.redhat.com systemd[1]: Failed to start OpenShift Node.
Aug 26 19:07:19 master.p1.rhc-ose.labs.redhat.com systemd[1]: Unit openshift-node.service entered failed state.
[root@master ~]# systemctl status docker
docker.service - Docker Application Container Engine
   Loaded: loaded (/usr/lib/systemd/system/docker.service; disabled)
  Drop-In: /usr/lib/systemd/system/docker.service.d
           └─docker-sdn-ovs.conf
   Active: failed (Result: exit-code) since Wed 2015-08-26 19:07:18 PDT; 8min ago
     Docs: http://docs.docker.com
  Process: 61799 ExecStart=/usr/bin/docker -d $OPTIONS $DOCKER_STORAGE_OPTIONS $DOCKER_NETWORK_OPTIONS $ADD_REGISTRY $BLOCK_REGISTRY $INSECURE_REGISTRY (code=exited, status=2)
 Main PID: 61799 (code=exited, status=2)
   CGroup: /system.slice/docker.service

Aug 26 19:07:18 master.p1.rhc-ose.labs.redhat.com docker[61799]: flag needs an argument: --add-registry
Aug 26 19:07:18 master.p1.rhc-ose.labs.redhat.com docker[61799]: See '/usr/bin/docker --help'.
Aug 26 19:07:18 master.p1.rhc-ose.labs.redhat.com systemd[1]: docker.service: main process exited, code=exited, status=2/INVALIDARGUMENT
Aug 26 19:07:18 master.p1.rhc-ose.labs.redhat.com systemd[1]: Failed to start Docker Application Container Engine.
Aug 26 19:07:18 master.p1.rhc-ose.labs.redhat.com systemd[1]: Unit docker.service entered failed state.
```

**Comment by detiber** [comment]
@etsauer you are probably hitting this bug: https://github.com/openshift/openshift-ansible/issues/514

**Comment by wshearn** [comment]
Merged #516 should be fixed now
